src/item.cpp

```diff
@@ -3965,6 +3984,19 @@ bool item::ammo_consume( int qty ) {
     return false;
 }
 
+itype_id item::ammo_current() const
+{
+    if( is_ammo() ) {
+        return typeId();
+    }
+
+    if( is_magazine() ) {
+        return contents.size() ? contents[0].type->id : "null";
```

Comment:
Logically equivalent however I agree `!contents.empty()` is more semantically correct